src/lv_misc/lv_txt_ap.c

```diff
@@ -125,16 +126,35 @@ uint32_t _lv_txt_ap_calc_bytes_cnt(const char * txt)
     return chars_cnt + 1;
 }
 
+uint32_t _lv_txt_lam_alef(uint32_t ch_curr, uint32_t ch_next)
```

Comment:
Can we change it to `static`?

Reply:
yes. i will change

Reply:
I changed the function to static, moved it to static section, adjusted return type to follow other code variable types and tested it.